_includes/use-cases/use-case-consumption-accept-incoming-request.md

```diff
@@ -0,0 +1,18 @@
+Attempts to accept an incoming request by it's Id.
+
+A [request](/integrate/data-model-overview#request) consists of [RequestItems](/integrate/data-model-overview#requestitem)
+and/or [RequestItemGroups](/integrate/data-model-overview#requestitemgroup). To accept the request
+the body has to contain a items array that indicates the decisions made for each request item.
+The decision are expressed through [ResponseItems](/integrate/data-model-overview#responseitem) whose
+structure is dependent on the corresponding RequestItem.
```

Comment:
This is not correct. When talking to the api's you answer the requestItems with so called `Parameters` (e.G. https://enmeshed.eu/integrate/requests-and-requestitems#authenticationrequestitem-response-parameters). These parameters are then translated to actual ResponseItems.

That means for accepting an AuthenticationRequestItem you send `{ "accept": true }` which will be translated to
an [AcceptResponseItem](https://enmeshed.eu/integrate/data-model-overview#acceptresponseitem).

Additionally I see some copy paste on these pages. Maybe it could be great to move this to another _include and then just reference it here.